bundles/org.openhab.binding.mystrom/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -11,6 +11,7 @@
 		<channels>
 			<channel id="switch" typeId="system.power"/>
 			<channel id="power" typeId="power-channel"/>
+			<channel id="ws" typeId="ws-channel"/>
```

Comment:
Perhaps it would be better to name this channel after its meaning rather than its internal unit. I have no really good suggestion, but to start the process: `energy-consumed-since-last-call`.